source/clear-linux/guides/maintenance/kernel-modules.rst

```diff
@@ -216,6 +217,4 @@ Related topic
 
 * :ref:`kernel-modules-dkms`
 
-
-.. _`on GitHub`: https://github.com/clearlinux/distribution
 .. _`mixer tool`: https://clearlinux.org/features/mixer-tool
```

Comment:
Please update to `:ref:mixer` on Ln 71 above.  Delete hyperlink at bottom. Thanks